src/prompt.c

```diff
@@ -127,6 +127,20 @@ prompt_default_handler(struct input *input, struct key *key)
 }
 
 static enum input_status
+prompt_script_handler(struct input *input, struct key *key)
```

Comment:
I am not sure if defining this is actually necessary. `read_prompt_incremental` allows to pass a `NULL` handler.